post_processing/pylbo/visualisation/eigenfunctions/eigfunc_handler.py

```diff
@@ -86,3 +106,60 @@ def _get_title(self, ef_name):
     def _mark_points_without_data_written(self):
         self._condition_to_make_transparent = "has_efs"
         super()._mark_points_without_data_written()
+
+    def _invert_continua(self, ds, ev_idx):
+        """
+        Calculates the locations of resonance with the continua for a specific
+        eigenmode.
+
+        Parameters
+        ----------
+        ef_idx : int
+            The number of the eigenvalue in the dataset.
+
+        Returns
+        -------
+        r_inv : dict
+            Dictionary of continua names and inverted resonance locations
+            (float, or None if not in domain).
+        labels : dict
+            Dictionary containing the corresponding labels to be printed when drawing
+            the locations of resonance.
+        """
+
+        CONTINUUM_LABELS = {
+            "slow-": r"$r(\Omega_S^-)",
+            "slow+": r"$r(\Omega_S^+)",
+            "alfven-": r"$r(\Omega_A^-)",
+            "alfven+": r"$r(\Omega_A^+)",
+            "thermal": r"$r(\Omega_T)",
+            "doppler": r"$r(\Omega_0)",
+        }
```

Comment:
```suggestion
```
Put these in the `continua.py` file and import them so they're consistent throughout, merge with `CONTINUA_NAMES` there

Reply:
Do you mean changing the CONTINUA_NAMES to these LaTeXised labels or something else? I planned to use these to show the continuum name at each location of resonance but ended up doing this by varying the linestyles.

Reply:
Yeah it's great to add the texised labels as well to be consistent. I meant that the `CONTINUUM_LABELS` variable here indirectly contains `CONTINUUM_NAMES` as well so you can merge the two in a dict or so and keep only one variable

Reply:
I created a dictionary of continua names and texnames if ever needed, but this may lead to clumsy coding in the return statement of `calculate_continua`. Feel free to improve.

Reply:
That's fine, we'll tweak it at some point when handling #131